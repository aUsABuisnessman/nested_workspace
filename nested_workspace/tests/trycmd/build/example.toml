args = ["build", "-vv"]

[bin]
name = "cargo"

[fs]
cwd = "../../../../example"
