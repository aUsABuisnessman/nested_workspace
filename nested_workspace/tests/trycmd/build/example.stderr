...
    Finished `dev` profile [unoptimized + debuginfo] target(s) in [..]s
